lib/puppet/provider/ec2_vpc/v2.rb

```diff
@@ -0,0 +1,94 @@
+require_relative '../../../puppet_x/puppetlabs/aws.rb'
+
+Puppet::Type.type(:ec2_vpc).provide(:v2, :parent => PuppetX::Puppetlabs::Aws) do
+  confine feature: :aws
+
+  mk_resource_methods
+  remove_method :tags=
```

Comment:
Just a note that in extending the test suite I've added tests demonstrating the tagging of resources.